configs/default/RUSH-BLADE_F7_HD.config

```diff
@@ -0,0 +1,113 @@
+# version
+# Betaflight / RUSHFCHD (RF7H) 4.2.1 Aug 13 2020 / 00:11:54 (caa0d683c) MSP API: 1.43
+
+# start the command batch
+batch start
```

Comment:
This should not have `batch` commands in it.

Reply:
This is my negligence, sorry!